users/ignoring.rst

```diff
@@ -140,3 +142,42 @@ all files and directories called "foo", ending in a "2" or starting with
   ``some/directory/`` matches the content of the directory, but not the
   directory itself. If you want the pattern to match the directory and its
   content, make sure it does not have a ``/`` at the end of the pattern.
+
+Ignoring files across all devices
+---------------------------------
+
+Although ``.stignore`` file is not synced between the devices, as mentioned
+previously, it's possible to take advantage of an ``#include`` pattern
+to create a single file that will hold shared ignoring patterns.
+
+Assuming the content of a file ``.stignore-shared`` is::
+
+    // Shared ignoring patterns
+    (?d).DS_Store
+    foo
+
+This file can be synced across all devices and helps with ignoring
+``.DS_Store`` if you sync a folder between devices where at least one is
+Mac OS and will also cause ``foo`` pattern being ignored on all devices.
```

Comment:
"which is useful if you sync a folder between devices where one of them is using Mac OS." 

This I am not sure of but (is it containing or is it starting?):

This will also cause any items containing foo pattern to be ignored on all devices.